scripts/index.js

```diff
@@ -13,7 +13,7 @@ const appConfig = __SUPERDESK_CONFIG__;
 import externalApps from 'external-apps';
 
 if (appConfig.features.useTansaProofing) {
-    require('apps/tansa');
+    setTimeout(require('apps/tansa'), 0);
```

Comment:
What is the effect of this? Perhaps an explanation why you did this would be helpful to others. Maybe a link where we can find more about this approach. I think `require.ensure` is for async, in WebPack.